onnxscript/ir/passes/common/shape_inference_test.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+from __future__ import annotations
+
+import unittest
+
+from onnxscript import ir
+from onnxscript.ir.passes.common import shape_inference
+
+
+class TestShapeInference(unittest.TestCase):
+    def test_shape_inference(self):
```

Comment:
maybe we could create a test case for model with initializers?

Reply:
Done, thanks!